lib/components/proxy-request-handler.js

```diff
@@ -1,11 +1,14 @@
 'use strict';
 
 var httpProxy = require('http-proxy');
+var _ = require('lodash');
 var contracts = require('../../api/contracts');
 
 var targetAddress = 'http://localhost:8080/';
 var proxy;
 
+var responseHandlers = [];
```

Comment:
all components are singletons, unless they are defined as a function
I didn't want to expose these as state.